uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -531,7 +535,21 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
     }
 
     private fun showTravelDetailsDialog(){
-        //To be implemented
+        val travelDetailsBottomSheet = CheckoutTravelDetailsBottomSheet().apply {
+            isFlight = presenter.getJourneyDetails()?.pickup?.details?.type == PoiType.AIRPORT
+            onValueChanged = {
+                bindTravelDetails(if(isFlight) PoiType.AIRPORT else PoiType.TRAIN_STATION, it)
+            }
+            flightInfo.isNotEmpty().let {
+                if(it)
```

Comment:
Redundant let closure, we should have a simple if(flightInfo.IsNotEmpty()) { initialValue = flightInfo }